pkg/adaptor/cloud/libvirt/types.go

```diff
@@ -26,9 +26,9 @@ type vmConfig struct {
 	mem          uint
 	rootDiskSize uint64
 	userData     string
-	metaData     string
-	ips          []netip.Addr
-	instanceId   string //keeping it consistent with sandbox.vsi
+	// metaData     string
```

Comment:
unintentional leftover ?

Reply:
Rest looks good to me. Thanks @Carsten-Leue

Reply:
@Carsten-Leue sorry It'll need a rebase again. And I think you'll need to remove the commented out variable as well and we should be good to merge this.

Reply:
@bpradipt I removed the line and tried by best to rebase correctly. Could you please take a fresh look.